src/main/java/org/jabref/gui/WelcomePage.java

```diff
@@ -0,0 +1,209 @@
+
+
+        addStartLabel();
+        items.setAlignment(Pos.CENTER);
+        welcomePageItems.getChildren().add(items);
+        welcomeBox.getChildren().add(welcomePageItems);
+    }
+
+    public void addStartLabel() {
+        Label start = new Label("Start:");
+        start.setStyle(headings);
+//        start.setFont(Font.font("Arial", FontWeight.BOLD, 30));
+
+        start.setTranslateX(start.getTranslateX() - 90);
+        items.getChildren().add(start);
+        items.setAlignment(Pos.CENTER_LEFT);
+        addNewLibraryButton();
+//        addOpenLibraryButton();
+        addConnectDataBaseButton();
+        addImportLibraryButton();
+    }
+
+    public void addNewLibraryButton() {
+        newLibraryLabel = new Label("New Library");
+        newLibraryLabel.setStyle(fontStyle);
+        newLibraryLabel.setOnMouseEntered(event -> {
+            newLibraryLabel.setStyle(hover);
+        });
+        newLibraryLabel.setOnMouseExited(event -> {
+            newLibraryLabel.setStyle(fontStyle);
+        });
+        newLibraryLabel.setAlignment(Pos.CENTER);
+        //VBox.setMargin(newLibraryLabel, new javafx.geometry.Insets(0, 0, 0, 20));
+        items.getChildren().add(newLibraryLabel);
+    }
+
+    public void addOpenLibraryButton() {
+        openLibrarylabel = new Label("Open Library");
+        openLibrarylabel.setStyle(fontStyle);
+        openLibrarylabel.setOnMouseEntered(event -> {
+            openLibrarylabel.setStyle(hover);
+        });
+        openLibrarylabel.setOnMouseExited(event -> {
+            openLibrarylabel.setStyle(fontStyle);
+        });
+        openLibrarylabel.setAlignment(Pos.CENTER);
+
+        items.getChildren().add(openLibrarylabel);
+    }
+
+    public void addConnectDataBaseButton() {
+        connectToDatabase = new Label("Connect to Shared Database");
+        connectToDatabase.setStyle(fontStyle);
+        connectToDatabase.setOnMouseEntered(event -> {
+            connectToDatabase.setStyle(hover);
+        });
+        connectToDatabase.setOnMouseExited(event -> {
+            connectToDatabase.setStyle(fontStyle);
+        });
+        connectToDatabase.setAlignment(Pos.CENTER);
+
+        items.getChildren().add(connectToDatabase);
+    }
+
+    public void addImportLibraryButton() {
+        importNewLib = new Label("Import into New Library");
+        importNewLib.setStyle(fontStyle);
+        importNewLib.setOnMouseEntered(event -> {
+            importNewLib.setStyle(hover);
+        });
+        importNewLib.setOnMouseExited(event -> {
+            importNewLib.setStyle(fontStyle);
+        });
+        importNewLib.setAlignment(Pos.CENTER);
+
+        items.getChildren().add(importNewLib);
+    }
+
+    public void runLabelTasks() {
+        newLibraryLabel.setOnMouseClicked(new EventHandler<MouseEvent>() {
+            @Override
+            public void handle(MouseEvent event) {
+                new NewDatabaseAction(JabRefGUI.getMainFrame(), preferencesService).execute();
```

Comment:
No direct call JabRefGUI.getMainFrame(). Please inject the frame in the constructor if you must.